### PR TITLE
Prepare minor 0.100.1 release

### PR DESCRIPTION
**Pull request by alejoe91**
@samuelgarcia please check!

### CODE REVIEW

File: doc/whatisnew.rst [hunk 1]
```diff
@@ -34,6 +35,12 @@ Release notes
   releases/0.9.1.rst
 
 
+Version 0.99.1
```
Comment (samuelgarcia) [inline doc/whatisnew.rst:38]:
```suggestion
Version 0.100.1
```

**Comment by samuelgarcia** [comment]
do not forget to:
 *  comment all the github dependencies in the pyproject.toml
 * put DEV=False in the init.py

**Comment by alejoe91** [comment]
That's already done

**Comment by samuelgarcia** [comment]
This should not be the case but that's ok.

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**